livox_ros_driver/livox_ros_driver/lddc.cpp

```diff
@@ -493,8 +510,7 @@ uint32_t Lddc::PublishImuData(LidarDataQueue *queue, uint32_t packet_num,
   uint64_t timestamp = 0;
   uint32_t published_packet = 0;
 
-  sensor_msgs::Imu imu_data;
-  imu_data.header.frame_id.assign(imu_frame_id_);
+  imu_data_.header.frame_id.assign(imu_frame_id_);

```

Comment:
Why'd you add the IMU sensor message as a class member?

Reply:
Because I'm modifying the covariances only once and to save some time setting some constant values on each loop, the message is defined as a class member.